app/views/users/edits.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="row">
+  <h2>Recent edits made by <%= link_to @user.username, user_page_path(@user) %></h2>
+</div>
+
+<div class="row">
+  <div id="user-edits-table-container" class="col-sm-12 nopadding">
+    <table class="table" id="user-edits-table">
+      <thead>
+	<tr>
+	  <th>Resource</th>
+	  <th>Action</th>
+	  <th>Time</th>
+	</tr>
+      </thead>
+      <tbody>
+	<% @edits.recent_edits_presenter.each do |edit| %>
```

Comment:
nice use of a presenter here. really simplifies the view logic. :)

Reply:
:)